pkg/solana/chain.go

```diff
@@ -221,6 +229,61 @@ func newChain(id string, cfg *config.TOMLConfig, ks loop.Keystore, lggr logger.L
 		lggr:        logger.Named(lggr, "Chain"),
 		clientCache: map[string]*verifiedCachedClient{},
 	}
+
+	if cfg.MultiNodeEnabled() {
+		chainFamily := "solana"
+
+		mnCfg := cfg.MultiNodeConfig()
+
+		var nodes []mn.Node[mn.StringID, *client.Client]
+
+		for i, nodeInfo := range cfg.ListNodes() {
+			rpcClient, err := client.NewClient(nodeInfo.URL.String(), cfg, DefaultRequestTimeout, logger.Named(lggr, "Client."+*nodeInfo.Name))
+			if err != nil {
+				lggr.Warnw("failed to create client", "name", *nodeInfo.Name, "solana-url", nodeInfo.URL.String(), "err", err.Error())
+				return nil, fmt.Errorf("failed to create client: %w", err)
+			}
+
+			newNode := mn.NewNode[mn.StringID, *client.Head, *client.Client](
+				mnCfg, mnCfg, lggr, *nodeInfo.URL.URL(), nil, *nodeInfo.Name,
+				i, mn.StringID(id), 0, rpcClient, chainFamily)
+
+			nodes = append(nodes, newNode)
+		}
+
+		multiNode := mn.NewMultiNode[mn.StringID, *client.Client](
+			lggr,
+			mn.NodeSelectionModeRoundRobin,
+			0,
+			nodes,
+			[]mn.SendOnlyNode[mn.StringID, *client.Client]{},
```

Comment:
Are send only RPCs relevant for Solana? I don't see them used anywhere currently, so I assumed there would be probably be none. This can change if we want to include them however.

Reply:
i think it's relevant if we want to configure reading from a single node and just sending txs to many nodes?